recipes-extended/netconfig/files/lo.1.network

```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=lo
+
+[Network]
```

Comment:
Why do you need empty [Network] and [Route] nodes?

Reply:
Sounds fair, nodes are removed.